### DISCUSSION TITLE
Canvas Get Stuck when using fabric.loadSVGFromString

### DISCUSSION THREAD

**Discussion by umanda** [fabricjs/fabric.js#7225] [Q&A]
When I load SVGs via `fabric.loadSVGFromString(svg)`, the Fabric Canvas gets stuck and ultimately the browser freezes. However, I don't observe any problems when I use `fabric.Image.fromURL(base64Svg)`. However, I can't utilize **base64** converted images since the `canvas.toSVG()` function returns an image, which I can't use for image conversion. (In my backend, I use an SVG to JPEG converter.)

Is there a way to resolve this problem? Your assistance is much appreciated.

**Comment by i-prikot** [comment]
Hi. Try using cache.

http://fabricjs.com/fabric-object-caching
http://fabricjs.com/svg-caching

**Reply by umanda** [reply]
Yes, I am using caching, it is helping me. Thank you verymuch

**Comment by umanda** [comment]
I have enabled cache and issue is fixed
